liquibase-standard/src/main/java/liquibase/changelog/ChangeSet.java

```diff
@@ -51,7 +51,7 @@
 /**
  * Encapsulates a changeSet and all its associated changes.
  */
-public class ChangeSet implements Conditional, ChangeLogChild {
+public class ChangeSet implements Conditional, ChangeLogChild, Comparable<ChangeSet> {
```

Comment:
Is there no other way to make sure flattening of `include/includeAll` is guaranteed? Updating a public class (and one important like this one) in this way, If I'm not wrong, could be a breaking change.

Reply:
We can remove the `Comparable` interface implementation and pass a `Comparator` directly to the `TreeSet` when needed if you want, but actually, I don't see any breaking changes in this.